src/Rifm.js

```diff
@@ -19,7 +20,10 @@ type Props = {|
 export const Rifm = (props: Props) => {
   const [, refresh] = React.useReducer(c => c + 1, 0);
   const valueRef = React.useRef(null);
-  const userValue = props.format(props.value);
+  const { replace } = props;
+  const userValue = replace
+    ? replace(props.format(props.value))
+    : props.format(props.value);
```

Comment:
props.format(props.value) can be reused to achieve better minified size

Reply:
At at least destructure format

Reply:
having gzip it s usually not true, in gzip this duplicate is just a back ref and size